packages/dotcom-ui-header/src/header.scss

```diff
@@ -62,4 +62,46 @@
 		$size: 32
 	);
   vertical-align: middle;
-}
\ No newline at end of file
+}
+
+.o-header__drawer-search-term {
+  min-width: 120px;
```

Comment:
**question:** Is this required? When looking at this in Storybook, I don't see it having an effect. It is best practice not to override Origami classes so trying to avoid it if we can.

Reply:
Unfortunately, the input width is not shrunk bellow certain width on resolutions between 981 and 1319 without it. I've tested on latest Chrome and Firefox on Windows and I've just double checked it again

![image](https://github.com/Financial-Times/dotcom-page-kit/assets/1334199/06d848b6-9800-43ee-828e-e748b0c65a3f)

This is closest explanation I've found https://stackoverflow.com/questions/42421361/input-button-elements-not-shrinking-in-a-flex-container

`min-width` solves the issue and I've choose 120px because this is a bit less than a minimum width I see this input can go. When I've played with it I've tried different approaches trying to avoid setting this - I've tried different html structure, css grid instead of flexbox, but they all involved much more html and css changes and the input didn't want to shrink anyway.

I've just tested an alternative solution with setting `width: 100%` instead of `min-width: 120px` as suggested in another SO comment. It is working and we will avoid the magic number. I will change it if you agree with that approach.

Reply:
Thank you for the thorough explanation! I agree that we should swap to the `width: 100%` rule instead 👍

Reply:
Updated. Here are few tests on my end

### With ASK FT button
#### Desktop
https://github.com/Financial-Times/dotcom-page-kit/assets/1334199/cada6ce6-0646-4bd2-81ae-4e53938a8a6d

#### Mobile
![image](https://github.com/Financial-Times/dotcom-page-kit/assets/1334199/096745c4-a59e-48e0-9a68-0db9328afd58)


### Without ASK FT button (keep unchanged)
#### Desktop
https://github.com/Financial-Times/dotcom-page-kit/assets/1334199/e50e72dd-92b4-4fa1-849e-2f2432e9f5a7

#### Mobile
![image](https://github.com/Financial-Times/dotcom-page-kit/assets/1334199/160a6f95-f7ac-4761-b821-ed01790d9243)